ddtrace/contrib/django/__init__.py

```diff
@@ -191,8 +205,9 @@
    tracer.set_tags({'env': 'production'})
 
    import my.custom.tracer
-   from ddtrace import Pin
+   from ddtrace import Pin, patch_all
    import django
+   patch_all()
```

Comment:
is `django` enabled by default now? does this need `patch_all(django=True)`?

Reply:
Yes it's enabled by default so left it out to avoid redundancy.

Reply:
perfect